pages/formester-vs-google-forms.vue

```diff
@@ -0,0 +1,419 @@
+<template>
+  <div>
+    <comparision-hero
+      heading="Switch to <span class='hglight'>Formester</span> and build beautiful online forms – The Best Alternative to Google Forms"
+      description="Tired of Google Forms's bland interface and lack of templates? Formester is here to unleash your inner form-building genius! Craft stunning, on-brand forms that capture attention, collect data, process orders and automate tasks like a pro. No coding required thanks to our Drag and Drop Interface!"
+      imgLink="formester-vs-google-form/formester-vs-google-form-hero-image"
+      imgAlt="Fomester vs Google Forms"
+    />
+
+    <section class="container my-5 py-5">
+      <div
+        class="d-flex flex-column align-items-center justify-content-center horizontal__padding"
+      >
+        <h2 class="section__heading">Formester vs Google Forms</h2>
+        <div class="d-flex flex-column flex-md-row gap-10 mt-4">
+          <div class="d-flex flex-column">
+            <nuxt-img
+              src="/comparision/formester-vs-google-form/formester-ss.png"
+              class="img-fluid screenshot formester-ss"
+            />
+            <p class="mt-3">
+              Formester is an online form builder that can be used to build
+              Forms, Surveys, Quizzes and Polls. It is a no-code tool with
+              robust data analytics and personalisation features. Formester has
+              AI powered Spam Protection along with multiple integrations to
+              automate workflow.
+            </p>
+          </div>
+          <div class="d-flex flex-column">
+            <nuxt-img
+              src="/comparision/formester-vs-google-form/google-form-ss.png"
+              class="img-fluid screenshot google-form-ss"
+            />
+            <p class="mt-3">
+              Google Forms is a free web form builder. It can be used to create
+              basic surveys and questionnaires with ease, suitable for casual
+              gatherings and simple data collection. Google forms is a vastly
+              popular tool but has limited design options, integrations and
+              functionality.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <comparision-table
+      heading="Feature Comparision"
+      description="Let’s have a look at the detailed feature comparison between Formester and Google Forms to help you pick the right option based on your form building needs."
+      :headers="['Fomester', 'Google Form']"
```

Comment:
```suggestion
      headers="['Fomester', 'Google Form']"
```
i think it will work without :

Reply:
nope it will not.